### ISSUE TITLE
Error: Could not find module `lodash` imported from `ember-calendar/models/day` upon installation

### ISSUE THREAD

**Issue by dmoon1221** [alphasights/ember-calendar#124]
In a new ember-cli project, I did the following:
1. `ember install ember-calendar`
2. Added `app/templates/index.hbs` and `app/controllers/index.js` as directed by the [Basic Usage](https://github.com/alphasights/ember-calendar#basic-usage) section
3. `ember s`

The result was that no content got rendered for `index.hbs` and I got the following console error message:
```
ember.debug.js:19818 Error: Could not find module `lodash` imported from `ember-calendar/models/day`
    at missingModule (loader.js:244)
    at findModule (loader.js:255)
    at Module.findDeps (loader.js:165)
    at findModule (loader.js:259)
    at Module.findDeps (loader.js:165)
    at findModule (loader.js:259)
    at Module.findDeps (loader.js:165)
    at findModule (loader.js:259)
    at Module.findDeps (loader.js:165)
    at findModule (loader.js:259)
defaultDispatch	@	ember.debug.js:19818
dispatchError	@	ember.debug.js:19797
invokeWithOnError	@	ember.debug.js:2200
flush	@	ember.debug.js:2256
flush	@	ember.debug.js:2380
end	@	ember.debug.js:2450
run	@	ember.debug.js:2564
join	@	ember.debug.js:2595
run.join	@	ember.debug.js:24493
(anonymous)	@	ember.debug.js:24556
mightThrow	@	jquery.js:3583
process	@	jquery.js:3651
```

This is despite seeing that `lodash` was added to `bower.json` and installed in `bower_components` upon installing `ember-calendar`.

My `ember -v`:
```
ember-cli: 2.13.2
node: 6.5.0
os: darwin x64
```

**Comment by willrax** [comment]
Thanks, I'll check this out today.

**Comment by willrax** [comment]
Ok, should be fixed in 0.4.1. Do you mind updating the package and then running `ember g ember-calendar` in your application?

**Comment by dmoon1221** [comment]
That fixed it! Thanks!
